assets/templates/area-summary.tmpl

```diff
@@ -0,0 +1,5 @@
+{{ $lang := .Language }}
```

Comment:
You're not using this assignment, please remove

Reply:
Sorry missed this after the last PR.